lib/default-props.js

```diff
@@ -31,7 +31,9 @@ var defaultProps = {
     variableWidth: false,
     vertical: false,
     // waitForAnimate: true,
-    afterChange: null
+    afterChange: null,
+    nextButton: null,
```

Comment:
@ryanbagwell  Can you change these names to "nextArrow" and "prevArrow"  to match the terminology of  slick carousel